### PR TITLE
[Site] Stop the ReactSvelteVue Mix

### PR DESCRIPTION
**Pull request by smnandre**
| Q             | A
| ------------- | ---
| Bug fix?      |no
| New feature?  | no
| License       | MIT

Remove a loooot of computation, script, events, by not calling React / Svelte / Vue files except on their demo.

### CODE REVIEW

File: ux.symfony.com/assets/app-vue.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import {registerVueControllerComponents} from "@symfony/ux-vue";
```
Comment (evertharmeling) [inline ux.symfony.com/assets/app-vue.js]: `import {registerVueControllerComponents} from '@symfony/ux-vue';`

Single quotes to keep it in line with the rest :)

Comment (smnandre) [reply ux.symfony.com/assets/app-vue.js]: Good catch, thank you

File: ux.symfony.com/importmap.php [hunk 2]
```diff
@@ -156,4 +168,20 @@
...
+    '@swup/theme' => [
+        'version' => '2.1.0',
+    ],
+    '@kurkle/color' => [
+        'version' => '0.3.2',
+    ],
+    'bootstrap/dist/css/bootstrap.min.css' => [
+        'version' => '5.3.2',
+        'type' => 'css',
+    ],
```
Comment (weaverryan) [inline ux.symfony.com/importmap.php]: Where did these new imports come from? Were they missing before? Or needed after upgrades?

**Review by weaverryan — COMMENTED** [review]
That’s a very interesting idea! Can you say more about the computation, events, etc that this saves? All of these controllers are lazy, so none of (for example) the react stuff should be loaded on a normal page. What am I not thinking about?

**Comment by smnandre** [comment]
⚠️  ~do not merge for now... i may have a problem with the loading order...~

Update: good

File: ux.symfony.com/importmap.php (hunk 2 above)
Comment (smnandre) [reply ux.symfony.com/importmap.php]: Good catch i'll check...

**Comment by smnandre** [comment]
This seems to work now with dedicated entrypoint "extending" app.js 

```js
// app-svelte.js 
import {registerSvelteControllerComponents} from '@symfony/ux-svelte';

import './app.js';

registerSvelteControllerComponents();
```

-- 

Info/note: 

We could also compose things, like that:

```twig
{% extends 'packageBase.html.twig' %}

{% block importmap %}
    {{ importmap(['app', 'app-react']) }}
{% endblock %}
```

```js
// app-react.js

import {registerReactControllerComponents} from '@symfony/ux-react';

registerReactControllerComponents();
```

File: ux.symfony.com/importmap.php (hunk 2 above)
Comment (smnandre) [reply ux.symfony.com/importmap.php]: Fixed (i must have made something i should not have 😅 )

**Comment by weaverryan** [comment]
Thanks Simon!

**Pull request merged**